### ISSUE TITLE
How to compute fusion matrix from CNN example?

### ISSUE THREAD

**Issue by tengerye** [tensorflow/tensorflow#2185]
In the tutorial 'Convolutional Neural Networks', tensorflow just computed the accuracy, but I wanted to compute the fusion matrix. I have trained the model, saved the variable and don't want to do it again. I am able to load the variables. Then three approaches crossed my mind: give new input to cifar10.inference; replace with new operations; directly output the prediction result. But I failed all of them, because: the inference does not take placeholder, no such operation could be implemented; multi-threaded code stopped me.

I posted a more detailed way of my trial in [stackoverflow](http://stackoverflow.com/questions/36960457/tensorflow-evaluate-with-fusion-matrix). Please help. And even will be more grateful if solutions are given more than one. 

Thanks in advance.

**Comment by yaroslavvb** [comment]
That question was a bit broad -- I think you were asking how to use skflow to compute confusion matrix on CIFAR input pipeline which possibly nobody has done before, hence no answers. I think if you break this task into smaller tasks, maybe only using TensorFlow, you'd have better luck. Closing this for now since it's not a bug or feature request.

**Comment by ilblackdragon** [comment]
Posted [an answer on SO](http://stackoverflow.com/questions/36960457/tensorflow-evaluate-with-fusion-matrix/36987469#36987469) - leaving here link in case people stumble this issue.
